### DISCUSSION TITLE
v0.3.6

### DISCUSSION THREAD

**Discussion by ilyes319** [ACEsuit/mace#517] [Announcements]
## What's Changed
* bugfix for stress loss by @JPDarby in https://github.com/ACEsuit/mace/pull/438
* change stress normalization + setup.cfg with numpy<2.0 by @ilyes319 in https://github.com/ACEsuit/mace/pull/467
* fix universal loss stress by default by @ilyes319 in https://github.com/ACEsuit/mace/pull/472
* Support for schedulefree optimizer by @RokasEl in https://github.com/ACEsuit/mace/pull/477
* Add Command-Line Argument for Specifying Wandb Directory by @nqhq-lou in https://github.com/ACEsuit/mace/pull/466
* Add DOI badge and automated release by @ElliottKasoar in https://github.com/ACEsuit/mace/pull/404
* Merge latest develop by @ilyes319 in https://github.com/ACEsuit/mace/pull/480
* remove DOI from badge by @ilyes319 in https://github.com/ACEsuit/mace/pull/482
* add finetuning to readme TOC by @ilyes319 in https://github.com/ACEsuit/mace/pull/485
* add support for hessian in calculator by @ilyes319 in https://github.com/ACEsuit/mace/pull/486
* prepare the model for hessian by @ilyes319 in https://github.com/ACEsuit/mace/pull/490
* BUG: Single atoms not isolated incorrectly removed from training set by @jameskermode in https://github.com/ACEsuit/mace/pull/492
* Fix bug with isolated atoms by @ilyes319 in https://github.com/ACEsuit/mace/pull/493
* correct pbc extension by @ilyes319 in https://github.com/ACEsuit/mace/pull/506
* change dynamo to optional by @ilyes319 in https://github.com/ACEsuit/mace/pull/510
* Simplify installation steps by @vue1999 in https://github.com/ACEsuit/mace/pull/514
* Develop by @ilyes319 in https://github.com/ACEsuit/mace/pull/515
* remove Literal for python >3.9 by @ilyes319 in https://github.com/ACEsuit/mace/pull/516

## New Contributors
* @JPDarby made their first contribution in https://github.com/ACEsuit/mace/pull/438
* @nqhq-lou made their first contribution in https://github.com/ACEsuit/mace/pull/466
* @ElliottKasoar made their first contribution in https://github.com/ACEsuit/mace/pull/404
* @jameskermode made their first contribution in https://github.com/ACEsuit/mace/pull/492

**Full Changelog**: https://github.com/ACEsuit/mace/compare/v0.3.5...v0.3.6

<hr /><em>This discussion was created from the release <a href='https://github.com/ACEsuit/mace/releases/tag/v0.3.6'>v0.3.6</a>.</em>
